deployment/query-exporter/config.yaml

```diff
@@ -0,0 +1,388 @@
+        FROM (
+          SELECT
+            schemaname, tablename, cc.reltuples, cc.relpages, bs,
+            -- Optimal table pages: +4 for item pointer, bs-20 for page header overhead
+            CEIL((cc.reltuples*((datahdr+ma-
+              (CASE WHEN datahdr%ma=0 THEN ma ELSE datahdr%ma END))+nullhdr2+4))/(bs-20::float)) AS otta,
+            COALESCE(c2.relname,'?') AS iname, 
+            COALESCE(c2.reltuples,0) AS ituples, 
+            COALESCE(c2.relpages,0) AS ipages,
+            -- Optimal index pages: datahdr-12 for index header, bs-20 for page header
+            COALESCE(CEIL((c2.reltuples*(datahdr-12))/(bs-20::float)),0) AS iotta
+          FROM (
+            SELECT
+              ma,bs,schemaname,tablename,
+              (datawidth+(hdr+ma-(case when hdr%ma=0 THEN ma ELSE hdr%ma END)))::numeric AS datahdr,
+              (maxfracsum*(nullhdr+ma-(case when nullhdr%ma=0 THEN ma ELSE nullhdr%ma END))) AS nullhdr2
+            FROM (
+              SELECT
+                schemaname, tablename, hdr, ma, bs,
+                SUM((1-null_frac)*avg_width) AS datawidth,
+                MAX(null_frac) AS maxfracsum,
+                hdr+(
+                  -- Null bitmap size: count/8 converts bits to bytes, +1 for bitmap header
+                  SELECT 1+count(*)/8
+                  FROM pg_stats s2
+                  WHERE null_frac<>0 AND s2.schemaname = s.schemaname AND s2.tablename = s.tablename
+                ) AS nullhdr
+              FROM pg_stats s, (
+                SELECT
+                  (SELECT current_setting('block_size')::numeric) AS bs,
+                  -- Tuple header size in bytes (23 bytes for PostgreSQL 14+)
+                  23 AS hdr,
+                  -- Memory alignment in bytes, we only run on Linux containers, so this is always 4
+                  4 AS ma
+                FROM (SELECT version() AS v) AS foo
+              ) AS constants
+              GROUP BY 1,2,3,4,5
+            ) AS foo
+          ) AS rs
+          JOIN pg_namespace nn ON nn.nspname = rs.schemaname
+          JOIN pg_class cc ON cc.relname = rs.tablename AND cc.relnamespace = nn.oid
+          LEFT JOIN pg_index i ON indrelid = cc.oid
+          LEFT JOIN pg_class c2 ON c2.oid = i.indexrelid
+        ) AS sml
+        WHERE schemaname NOT LIKE 'pg_%' AND schemaname != 'information_schema'
+      )
+      SELECT COUNT(*) AS pg_table_bloat_count
+      FROM bloat_info
+      WHERE table_bloat > :table_bloat_threshold OR index_bloat > :index_bloat_threshold;
+
+  # Query 2: Detailed table and index bloat information
+  table_bloat_details:
+    interval: 300  # Run every 5 minutes
+    databases: [openremote]
+    metrics: [pg_table_bloat_ratio, pg_table_bloat_bytes, pg_table_bloat_wasted_mb]
+    parameters:
+      - table_bloat_threshold: !env TABLE_BLOAT_THRESHOLD
+        index_bloat_threshold: !env INDEX_BLOAT_THRESHOLD
+    sql: |
```

Comment:
Just ran this query on two of our DBs and I get back values in the 50s what is expected from this?